### DISCUSSION TITLE
Extension to show the status bar for DataGrid

### DISCUSSION THREAD

**Discussion by gulshan-verma** [dotnet/DataGridExtensions#113] [Ideas]
It would be wonderful to incorporate extension to add a status bar at the bottom of the DataGrid, indicating the current count of rows out of the total.

**Comment by tom-englert** [comment]
Isn't that a bit too trivial to make an extension for it?

**Reply by gulshan-verma** [reply]
I share the same sentiment. It's more of a nice-to-have feature, so no need to fret. By the way, your extension is truly fantastic, it's making life a lot easier. ⭐⭐⭐⭐⭐
